docs/documentation/features/frameworkUtilities.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: documentation
+---
+
+# Framework Utilities
+
+In this chapter useful services/utilities of the Eclipse SmartHome project are described. 
+
+## Network Address Service
+
+A user can configure his default network address via Paper UI under `Configuration -> System -> Network Settings`.
+The `NetworkAddressService` is an OSGi service that can be used like any other OSGi service by adding a service reference to it.
```

Comment:
This should be the first sentence of the paragraph. Maybe also mention that it is a configurable service, mentioning its service id. The Paper UI example above is then just one way - in general anything that uses ConfigurationAdmin can be used to configure the service.